config/env_spec_request.go

```diff
@@ -86,6 +141,88 @@ type jwtResult struct {
 	err    error
 }
 
+func (e *EnvironmentSpecRequest) parseRequestVariables(pathTemplate *transform.Template, opPath, queryString string) *requestVariables {
+
+	vars := &requestVariables{
+		path:    pathTemplate.Extract(opPath),
+		headers: e.Request.Attributes.Request.Http.Headers,
+		request: map[string]string{},
+		query:   map[string]string{},
+	}
+
+	vars.request[RequestPath] = opPath
+	vars.request[RequestQuerystring] = queryString
```

Comment:
Why do we need to store this?

Reply:
The "request.path" variable is just the path part, the "request.querystring" is just the query string part. The idea is to give users options to deal with the larger parts of the path instead of just the individually parsed pieces. For example, I might not want to be able to specify the entire request query string when I design the API... so this lets me just use the whole thing.

Reply:
I see. Thanks.